vendor/k8s.io/kubernetes/pkg/kubelet/dockershim/helpers.go

```diff
@@ -340,15 +343,44 @@ func getSecurityOptSeparator(v *semver.Version) rune {
 
 // ensureSandboxImageExists pulls the sandbox image when it's not present.
 func ensureSandboxImageExists(client dockertools.DockerInterface, image string) error {
+	dockerCfgSearchPath := []string{"/.docker", filepath.Join(os.Getenv("HOME"), ".docker")}
```

Comment:
Should this be "./.docker" ?

Or is looking in the top of the file system in /.docker intentional here?

Reply:
I didn't really notice that before.  Might ask on the upstream PR why this is.

What trouble are you having with the containerized node?  I figured it would Just Work (tm) since /var/lib/origin is bind mounted from the host into the containerized node.

Reply:
@michaelgugino see https://github.com/kubernetes/kubernetes/pull/46236#discussion_r138408509